apps/meteor/client/views/omnichannel/analytics/AnalyticsPage.tsx

```diff
@@ -90,12 +104,30 @@ const AnalyticsPage = () => {
 							w='66%'
 							h='100%'
 							chartName={chartName}
-							departmentId={department?.value}
+							departmentId={
+								(
+									department as {
```

Comment:
The fact that you need to do this means there's something wrong with the department type or some validation missing

Can it be {} | null instead of the 3-way union it is?